x-pack/legacy/plugins/uptime/public/components/functional/alerts/uptime_alerts_context_provider.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { AlertsContextProvider } from '../../../../../../../plugins/triggers_actions_ui/public';
+import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
+
+interface Props {
+  children: any;
+}
+
+export const UptimeAlertsContextProvider = ({ children }: Props) => {
```

Comment:
instead of using a children props, i usually use React.FC type

Reply:
💯 makes total sense 
resolved https://github.com/elastic/kibana/pull/57919/commits/588483269d5d60795dd1ad6a880e35a6af95554d